app/src/main/res/values/strings.xml

```diff
@@ -935,4 +936,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_pedestrian_crossing_island">"Does this pedestrian crossing have an island?"</string>
     <string name="quest_surface_detailed_answer_impossible_title">Describe surface</string>
     <string name="open_url">Open URL</string>
+    <string name="quest_dietType_kosher_name_title">Does %s have kosher products in the offer?</string>
```

Comment:
```suggestion
    <string name="quest_dietType_kosher_name_title">Does %s offer kosher (%s) products?</string>
```
Maybe add the Hebrew word [כשרות](https://he.m.wikipedia.org/wiki/%D7%9B%D7%A9%D7%A8%D7%95%D7%AA) in brackets behind the translation? (Also requires to override `getTitleArgs`)